x-pack/plugins/alerting/server/alerts_service/lib/set_alerts_to_untracked.test.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  ElasticsearchClientMock,
+  elasticsearchServiceMock,
+  loggingSystemMock,
+} from '@kbn/core/server/mocks';
+import { setAlertsToUntracked } from './set_alerts_to_untracked';
+
+let clusterClient: ElasticsearchClientMock;
+let logger: ReturnType<typeof loggingSystemMock['createLogger']>;
+
+describe('setAlertsToUntracked()', () => {
```

Comment:
can we get unit tests or functional tests for `ensureAuthorized` is true returning a siem consumer or a consumer that the user doesn't have access to?